examples/transloadit/index.html

```diff
@@ -27,102 +27,107 @@
       #logo { height: 1em; vertical-align: middle; }
     </style>
     <main>
-      <h1>Robodog <img src="https://transloadit.edgly.net/assets/images/logo-small.svg" alt="Transloadit" id="logo"> playground</h1>
+      <h1>Uppy Transloadit <img src="https://transloadit.edgly.net/assets/images/logo-small.svg" alt="Transloadit" id="logo"> playground</h1>
       <p>
-        This page contains small examples for every API offered by the Robodog library. Please see the <a href="https://github.com/transloadit/uppy/tree/main/examples/transloadit">Github repository</a> for the source code.
-
+        This page contains small examples for different ways you can use Uppy with Transloadit. Please see the <a href="https://github.com/transloadit/uppy/tree/main/examples/transloadit">Github repository</a> for the source code.
+      </p>
       <hr>
-      <h2>robodog.form()</h2>
+      <h2>Form</h2>
 
       <p>
-        The form API allows you to easily send files through Transloadit's encoding backend. When the user submits the form, any files are uploaded to Transloadit. The form data is then sent to your own backend, with additional data about the Transloadit Assemblies that were started.
-
+        The form API allows you to easily send files through Transloadit’s encoding backend. When the user submits the form, any files are uploaded to Transloadit. The form data is then sent to your own backend, with additional data about the Transloadit Assemblies that were started.
+      </p>
       <form id="test-form" method="post" action="http://localhost:9967/test">
-        <p><strong>leave a message</strong>
+        <p><strong>leave a message</strong></p>
         <p>
           <label>name:
             <input type="text" name="name">
           </label>
+        </p>
         <p>
           <label>message: <br>
             <textarea name="message"></textarea>
           </label>
-
+        </p>
         <p>
           <label>
             attachments:
-            <input type="file" name="files" multiple>
+            <!-- <input type="file" name="files" multiple> -->
+            <strong id="uppy-form-selected-files"></strong>
+            <button type="button" id="uppy-select-files">Select Files</button>
           </label>
           <div class="progress"></div>
-
+        </p>
         <p>
           <button type="submit">
             Upload
           </button>
-
+        </p>
           <span class="error"></span>
       </form>
 
       <hr>
-      <h2>robodog.form() with dashboard</h2>
+      <h2>Form with inline Dashboard</h2>
```

Comment:
I think I would keep referencing `robodog.form`, as this is meant to be a migration example. Same below.